java/core/src/main/java/sleeper/core/record/process/status/ProcessRuns.java

```diff
@@ -44,9 +44,9 @@ public static ProcessRuns fromRecordsLatestFirst(List<ProcessStatusUpdateRecord>
             ProcessStatusUpdateRecord record = recordList.get(i);
             String taskId = record.getTaskId();
             ProcessStatusUpdate statusUpdate = record.getStatusUpdate();
-            if (statusUpdate instanceof ProcessRunStartedUpdate) {
+            if (statusUpdate.isStartOfRun()) {
                 ProcessRun.Builder builder = ProcessRun.builder()
-                        .startedStatus((ProcessRunStartedUpdate) statusUpdate)
+                        .statusUpdates(statusUpdate)
```

Comment:
This wraps the status update in a list. It might be clearer if we use a method signature that explicitly just takes one update.